### ISSUE TITLE
Mac crash in Gitter Realtime

### ISSUE THREAD

**Issue by mydigitalself** [gitterHQ/desktop#48]
Silent crash today:

```
[69716:0323/125932:INFO:CONSOLE(28)] ""Caught exception: TypeError: undefined is not a function"", source: app://gitter/index.js (28)
[69716:0323/125932:INFO:CONSOLE(29)] ""TypeError: undefined is not a function\n    at instance.close (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/node_modules/websocket-extensions/lib/websocket_extensions.js:142:20)\n    at instance._shutdown (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/driver/hybi.js:299:22)\n    at instance._emitFrame (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/driver/hybi.js:419:12)\n    at instance.parse (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/driver/hybi.js:144:18)\n    at instance.parse (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/driver/client.js:57:58)\n    at IO.write (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/streams.js:80:16)\n    at TLSSocket.ondata (_stream_readable.js:533:20)\n    at TLSSocket.EventEmitter.emit (events.js:104:17)\n    at readableAddChunk (_stream_readable.js:156:16)\n    at TLSSocket.Readable.push (_stream_readable.js:123:10)"", source: app://gitter/index.js (29)
```

**Comment by trevorah** [comment]
```
TypeError: undefined is not a function
    at instance.close (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/node_modules/websocket-extensions/lib/websocket_extensions.js:142:20)
    at instance._shutdown (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/driver/hybi.js:299:22)
    at instance._emitFrame (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/driver/hybi.js:419:12)
    at instance.parse (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/driver/hybi.js:144:18)
    at instance.parse (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/driver/client.js:57:58)
    at IO.write (/Applications/Gitter.app/Contents/Resources/app.nw/node_modules/gitter-realtime-client/node_modules/gitter-faye/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/streams.js:80:16)
    at TLSSocket.ondata (_stream_readable.js:533:20)
    at TLSSocket.EventEmitter.emit (events.js:104:17)
    at readableAddChunk (_stream_readable.js:156:16)
    at TLSSocket.Readable.push (_stream_readable.js:123:10)
```

**Comment by suprememoocow** [comment]
Looking into this now. Quite possibly related to this (non) issue: https://github.com/faye/websocket-extensions-node/issues/2

**Comment by suprememoocow** [comment]
Wierd. `pipeline.js` seems to have been left behind after a npm upgrade. Will add a new issue to deleting node_modules before starting the packaging process.
